packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -164,6 +166,20 @@ export function buildRootObject() {
       return zcfMint;
     };
 
+    function updateNotiferFrom(updater, notifier, nextCount = undefined) {
```

Comment:
Can you just use `updateFromNotifier` in `packages/notifier`?

Reply:
yes, of course.